core/model/src/main/kotlin/remote/response/student/GetMyInformationResponse.kt

```diff
@@ -0,0 +1,16 @@
+package remote.response.student
+
+import java.util.UUID
+
+data class GetMyInformationResponse(
+    val id: UUID,
+    val currentPoint: Int,
+    val cumulatePoint: Int,
+    val user: User,
```

Comment:
```suggestion
    val user: User
```
필요 없는 `,`는 지워주시는 게 좋을 것 같아요! 다른 부분도요 ~

Reply:
이부분은 제가 알기로 마무리 짓는 data class 필드는 , 으로 마무리 해줘야 하는걸로 알고 있는데 한번 알아보겠습니다.

Reply:
일관성 유지를 이유로 모든 필드 뒤에 쉼표를 추가하면 새로운 필드를 추가하거나 순서를 변경할 때 코드 변경의 범위를 줄일 수 있어 모든 필드가 쉼표로 구분되므로 한줄의 변경 사항이 생기더라도 코드 변경의 한 줄에만 표시되어 가독성이 좋다고 생각이 됩니다. 어떻게 생각하시나요?

Reply:
@minStone-dev @yeongun130 같이 컨벤션 정해보시는 게 좋을 것 같네요 ~ (제 생각엔 없는 게 더 나을 것 같습니다!)

Reply:
저도 없는게 좋아 보입니다

Reply:
의논 결과 코틀린 공식 컨벤션에 따라 마지막 필드에는 , 를 추가하기로 했습니다.